src/components/Person.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {/* {age !== 0 ? (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    ) : (
+      null
+    )} */}
+    {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    )}
+
+    {isMarried ? (
+      <p className="Person__partner">
+        {'My '}
+        {sex === 'm'
+          ? 'wife\'s'
+          : 'husband\'s'
+        }
+        {` name is ${partnerName}`}
+      </p>
+    ) : (
+      'I am not married'
+    )
+
+      }
```

Comment:
It's better to create separate variable before return statement where you can store the result of inner conditional statement (checking sex === 'm')
```suggestion
    {isMarried ? (
      <p className="Person__partner">
        {`My ${partner}'s name is ${partnerName}`}
      </p>
    ) : (
      'I am not married'
    )

      }
```